Makefile

```diff
@@ -482,12 +482,12 @@ copy-docs:
 	@if [ ${ORDER} -ne 0 ]; then \
 		echo "sort: ${ORDER}" >> ${DST}; \
 		echo "weight: ${ORDER}" >> ${DST}; \
-		echo "menu:\n  docs:\n    parent: 'victoriametrics'\n    weight: ${ORDER}" >> ${DST}; \
+		echo -e "menu:\n  docs:\n    parent: 'victoriametrics'\n    weight: ${ORDER}" >> ${DST}; \
```

Comment:
this breaks compatibility with macos

have you considered using `printf` instead?

Reply:
Thanks! It fixed in the next commit.